tests/multi_partitions_setup.rs

```diff
@@ -0,0 +1,110 @@
+use samsa::prelude;
+use samsa::prelude::protocol::produce::request::Attributes;
+use samsa::prelude::{protocol, BrokerConnection, Error, KafkaCode, TcpConnection};
+use std::collections::HashMap;
+
+mod testsupport;
+
+const CLIENT_ID: &str = "multi partition read and write test";
+const CORRELATION_ID: i32 = 1;
+const NUMBER_OF_PARTITIONS: i32 = 10;
+
+#[tokio::test]
+async fn multi_partition_writing_and_reading() -> Result<(), Box<Error>> {
```

Comment:
For this test, lets repeat what is happening in Read and Write 1m messages with Multiple Parittions, but lets make it read and write 5 messages instead of 1M. That will be easier than building the low level requests